src/app/core/store/collections/collections.action.ts

```diff
@@ -66,6 +66,10 @@ export enum CollectionsActionType {
   CREATE_ATTRIBUTE_SUCCESS = '[Collections] Create Attribute :: Success',
   CREATE_ATTRIBUTE_FAILURE = '[Collections] Create Attribute :: Failure',
 
+  CREATE_ATTRIBUTES = '[Collections] Create Attributes',
```

Comment:
Do we still need actions for creating a single attribute after this new one was added?